.github/workflows/template-build.yml

```diff
@@ -112,6 +112,7 @@ jobs:
           GOOGLE_RECAPTCHA_KEY: ${{ secrets.GOOGLE_RECAPTCHA_KEY }}
           GOOGLE_RECAPTCHA_SITE_KEY: ${{ steps.KeyVaultSecrets.outputs.GoogleRecaptchaSiteKey }}
           KEY_VAULT: ${{ env.KEY_VAULT }}
+          NEXT_PUBLIC_APP_INSIGHT: ${{ env.NEXT_PUBLIC_APP_INSIGHT }}
```

Comment:
can we use clearer environment variable names... it isn't clear whether this is an resource name, endpoint, instrumentation key, or connection string